cssdb.json

```
@@ -526,6 +526,15 @@
       }
     ]
   },
+  {
+    "id": "prefers-reduced-motion",
```

Could we append `-query` to media query features? We do the same for functions and properties and pseudo classes.

```suggestion
    "id": "prefers-reduced-motion-query",
```